bpf/types.h

```diff
@@ -140,6 +140,8 @@ typedef struct additional_metrics_t {
     } translated_flow;
     u16 eth_protocol;
     u8 network_events_idx;
+    bool flow_encrypted;
+    u8 flow_encrypted_ret;
```

Comment:
if flow_encrypted is entirely derived from flow_encrypted_ret, maybe we can only keep the ret, and leave the interpretation to the user space? (as you prefer, I'm fine either way)

Reply:
hmmm I guess there's the extra case where IPSec isn't involved so flow_encrypted==false and flow_encrypted_ret==0 which we need to account for, we couldn't do that with just flow_encrypted_ret. OK so forget about my previous comment :-)

Reply:
sorry I am confused if both fields are default this is the no ipsec case right ?

Reply:
yes - no problem here